templates/addons/azure-cni-v1.yaml

```diff
@@ -50,7 +50,7 @@ spec:
         - /etc/cni/net.d/10-azure.conflist
         command:
         - /dropgz
-        image: mcr.microsoft.com/containernetworking/cni-dropgz:v0.0.4
+        image: mcr.microsoft.com/containernetworking/azure-cni:v1.5.16
```

Comment:
I was a bit confused here, because `v1.15.16` is a tag but not a GitHub release.

Reply:
Likewise, did the image name move from `cni-dropgz` to `azure-cni` as well?

Reply:
> Likewise, did the image name move from `cni-dropgz` to `azure-cni` as well?

@Jont828 Yes, that image is deprecated and this PR is to migrate CAPZ to the new installer image. If you check the description the linked issue there has more details.